https://github.com/python-llfuse/python-llfuse/commit/8c0160545ce01b3624c8ecad669f06fb6d22d2f6

From 8c0160545ce01b3624c8ecad669f06fb6d22d2f6 Mon Sep 17 00:00:00 2001
From: Thomas Waldmann <tw@waldmann-edv.de>
Date: Sun, 21 May 2023 01:31:38 +0200
Subject: [PATCH] remove Cython version check, fixes #81

distutils Version classes are deprecated.

I decided to remove the Cython version check completely,
it is not as simple as checking a minimum version anyway.

E.g. we can work with Cython 0.29.x and also 3.0.0b2,
but earlier 3.0.0 releases are of unknown compatibility.

Also, latest Python 3.12-dev versions require 0.29.34,
while older Pythons are happy with stuff compiled by earlier
0.29.x releases.
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,6 @@
 
 import setuptools
 from setuptools import Extension
-from distutils.version import LooseVersion
 
 basedir = os.path.abspath(os.path.dirname(__file__))
 sys.path.insert(0, os.path.join(basedir, 'util'))
@@ -218,11 +217,7 @@ def run(self):
                                               stderr=subprocess.STDOUT)
         except OSError:
             raise SystemExit('Cython needs to be installed for this command')
-
-        hit = re.match('^Cython version (.+)$', version)
-        if not hit or LooseVersion(hit.group(1)) < "0.29":
-            # in fact, we need a very recent Cython version to support the latest pythons
-            raise SystemExit('Need Cython 0.29 or newer, found ' + version)
+        print(f"Using {version.strip()}.")
 
         cmd = ['cython', '-Wextra', '--force', '-3', '--fast-fail',
                '--directive', 'embedsignature=True', '--include-dir',

